src/client/storage/uploader.ts

```diff
@@ -6,6 +6,14 @@ import {promises as fs} from 'fs';
 import path from 'path';
 import config from 'config';
 import logger from '../../core/log';
+import Arweave from 'arweave';
+
+const arweave = Arweave.init({
```

Comment:
We already have a set up arweave client in the `storage` module. Is it possible to reuse it by creating a sort of `getDataByTxId` in the `storage` and use it  within the `chunkValidatorLoop` instead?